dart-impl/base/CMakeLists.txt

```diff
@@ -120,6 +120,11 @@ set_target_properties(
   COMPILE_FLAGS ${ADDITIONAL_COMPILE_FLAGS}
 )
 
+set_target_properties(
+  ${DASH_DART_BASE_LIBRARY}
+  PROPERTIES POSITION_INDEPENDENT_CODE TRUE
+)
+
```

Comment:
Fine with me, couldn't see a performance impact significant to stick out of the noise in my experiments. That might be different on 32bit systems, though (in case anyone still uses them for real).